app/models/dojo.rb

```diff
@@ -13,6 +13,10 @@ class Dojo < ActiveRecord::Base
 
   before_create :add_creator_of_the_dojo_as_a_participant
 
+  def to_param
```

Comment:
Eu curto a ideia de evitar adicionar gems. Mas sobrescrevendo o to_param, `Dojo.find(params[:id])` funciona porque o id é a primeira parte da string gerada. 

Não sei bem se gosto disso ou não. :confused: 

Usando o friendly_id poderiamos até nem ter id na url, apenas o local. Em contrapartida, sobrescrever o to_param do jeito que vc fez deixa tudo muito simples, apesar de depender do id no início.

O que acha @m3nd3s?